pyuvsim/simsetup.py

```diff
@@ -131,7 +131,7 @@ def create_mock_catalog(time, arrangement='zenith', array_location=None, Nsrcs=N
     if array_location is None:
         array_location = EarthLocation(lat='-30d43m17.5s', lon='21d25m41.9s',
                                        height=1073.)
-    freq = (150e6 * units.Hz)
+    freq_array = (np.array(150e6) * units.Hz)
```

Comment:
Sorry to nitpick, but this will make `freq_array` have shape `()`, which can cause problems for indexing. This should still work since SkyModel runs `atleast_1d` on the input.

Reply:
how should I fix it? ` freq_array = np.array([150e6]) * units.Hz`?

Reply:
That should work, but if you've already fixed it it shouldn't matter.

Reply:
That should work, but if you've already fixed it then it's fine.